app/demo-geant-integration/HepMC3Reader.cc

```diff
@@ -51,8 +78,9 @@ void HepMC3Reader::GeneratePrimaryVertex(G4Event* g4_event)
     input_file_->read_event(gen_event);
     CELER_ASSERT(!input_file_->failed());
 
-    CELER_LOG_LOCAL(status)
-        << "Reading HepMC3 event " << gen_event.event_number();
+    CELER_LOG_LOCAL(info) << "Converting " << gen_event.particles().size()
```

Comment:
Should the `lock_guard` be applied to the rest of code (of course, don't harm) or only to around `read_event(gen_event)`?

Reply:
Definitely could be restricted to that. Good point!